rct229/rulesets/ashrae9012019/section19/section19rule22.py

```diff
@@ -0,0 +1,56 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.rulesets.ashrae9012019.data.schema_enums import schema_enums
+from rct229.utils.assertions import getattr_
+
+ENERGY_RECOVERY_OPERATION = schema_enums["EnergyRecoveryOperationOptions"]
+
+
+class Section19Rule22(RuleDefinitionListIndexedBase):
+    """Rule 22 of ASHRAE 90.1-2019 Appendix G Section 19 (HVAC - General)"""
+
+    def __init__(self):
+        super(Section19Rule22, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section19Rule22.HVACRule(),
+            index_rmr="baseline",
+            id="19-22",
+            description="Baseline systems modeled with exhaust air energy recovery shall allow bypass or control heat recovery system to permit air economizer operation.",
+            ruleset_section_title="HVAC - General",
+            standard_section=" Section G3.1.2.2",
+            is_primary_rule=True,
+            list_path="$.ruleset_model_instances[0].buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    class HVACRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section19Rule22.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                required_fields={
+                    "$": ["fan_system"],
+                },
+            )
+
+        def is_applicable(self, context, data=None):
+            hvac_b = context.baseline
+            fan_sys_b = hvac_b["fan_system"]
+
+            return fan_sys_b.get("air_energy_recovery") is not None
+
+        def get_calc_vals(self, context, data=None):
+            hvac_b = context.baseline
+            fan_sys_b = hvac_b["fan_system"]
+
+            ER_operation = getattr_(
+                fan_sys_b,
+                "Fan System",
+                "air_energy_recovery",
+                "energy_recovery_operation",
+            )
```

Comment:
I don't think we should use `getattr_` here because we do not want to raise exception to undetermined results due to missing `air_energy_recovery` or `air_energy_recovery.energy_recovery_operation`
Instead I suggest to use `find_one("$.air_energy_recovery.energy_recovery_operation", fan_sys_b)`

the `er_operation` equals to `None` if data is missing, which in the `rule_check` funciton, it will return `False`.